patches/minecraft/net/minecraft/world/World.java.patch

```diff
@@ -857,12 +857,17 @@
          }
      }
  
-@@ -2958,7 +3175,7 @@
+@@ -2954,11 +3171,11 @@
+         }
+         else if (iblockstate1.func_185904_a() == Material.field_151594_q && p_190527_1_ == Blocks.field_150467_bQ)
+         {
+-            return true;
++            return !net.minecraftforge.event.ForgeEventFactory.onBlockPlace(p_190527_5_, net.minecraftforge.common.util.BlockSnapshot.getBlockSnapshot(this, p_190527_2_), p_190527_4_, null).isCanceled();
```

Comment:
This change causes PlaceEvent to execute twice. Certain onItemUse calls already fire the event.